include/zephyr/bluetooth/audio/audio.h

```diff
@@ -1842,13 +1842,15 @@ int bt_audio_discover(struct bt_conn *conn,
  *  @param stream Stream object being configured
  *  @param ep Remote Audio Endpoint being configured
  *  @param codec Codec configuration
+ *  @param ase_id ID of the ASE being configured
  *
  *  @return Allocated Audio Stream object or NULL in case of error.
  */
 int bt_audio_stream_config(struct bt_conn *conn,
 			   struct bt_audio_stream *stream,
 			   struct bt_audio_ep *ep,
-			   struct bt_codec *codec);
+			   struct bt_codec *codec,
+			   uint8_t ase_id);
```

Comment:
I had to use `ase_get`, because `ascs->ases` is empty before initiating Codec Config, but I'm not sure if I can freely use it?

I may have misunderstood how to use `bt_audio_ep_info`, is it ok as it is now?